awx/main/utils/handlers.py

```diff
@@ -21,27 +22,31 @@ def _connect_unixsocket(self, address):
         super(RSysLogHandler, self)._connect_unixsocket(address)
         self.socket.setblocking(False)
 
+    def handleError(self, record):
```

Comment:
Just curious... why is this camel case?

Reply:
Whoever wrote this module in Python liked camel-case 🤷 

https://github.com/python/cpython/blob/master/Lib/logging/__init__.py#L1000

camelCase isn't too uncommon in older parts of cpython's source code.

Reply:
Better answer:  this Python module probably predates PEP8:

https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables